tests/test_main.py

```diff
@@ -147,7 +148,20 @@
                             'solvent': False}],
                'species_constraints': None,
                }
-
+rmg_minimal_defaults = rmg_minimal.copy()
+rmg_minimal_defaults['options'] = {'seed_name': 'Seed',
```

Comment:
Could you explain why we need to copy and add? Can't we add this block to the original `rmg_minimal`?

Reply:
I assumed it was intentional to leave the RMG options block as `None` as you'd defined it so make sure that the schema correctly assigns the defaults. So the updated dictionary `rmg_minimal_defaults` has the values after schema updates the dictionary so that the assert statement is True. Did you want me to change the line of `'options': None,` to reflect my copy and add so it has the arguments from `rmg_minimal_defaults`?

Reply:
I see. You're right. No need to add additional modifications.